spec/models/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(author_id: 1, title: 'title', text: 'text', comments_counter: 0, likes_counter: 0) }
+
+  before { subject.save }
+
+  it 'new post should be saved in the database' do
```

Comment:
- [ ] As per project requirements, your are to add units tests for all your models. At the moment you have tests just for the user and post model. I understand that you don't have any validations in the comment and like models, but you can at least add the test of new comment and new line being saved in the database.
![image](https://github.com/Stephen-Adom/rails_blog/assets/62849872/c095e4f3-6dfe-4d40-a36e-98d8a0a4ce6b)
![image](https://github.com/Stephen-Adom/rails_blog/assets/62849872/36969f0d-81e3-4e43-b0da-d2f606ec5871)

Reply:
This has been done well